### PR TITLE
[#281]调试器首页机器子项页点检问题修改

### LINKED ISSUE #281: 调试器首页优化点检问题修复

### PR DESCRIPTION
**Pull request by charles-chai**
fix #281

### CODE REVIEW

**Review by xiongliangoppo120 — APPROVED** [review]

**Review by duanchaodc — APPROVED** [review]

**Review by jianbo-oppo — APPROVED** [review]

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurLinearLayout.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022, hapjs.org. All rights reserved.
```
Comment (chenyinli) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurLinearLayout.java:2]: copyright

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurRelativeLayout.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022, hapjs.org. All rights reserved.
```
Comment (chenyinli) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurRelativeLayout.java:2]: copyright

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurShadowButton.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022, hapjs.org. All rights reserved.
```
Comment (chenyinli) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurShadowButton.java:2]: copyright

File: debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurShadowDrawable.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2022, hapjs.org. All rights reserved.
```
Comment (chenyinli) [inline debug/shell/android/app-impl/src/main/java/org/hapjs/debugger/widget/BlurShadowDrawable.java:2]: copyright

**Review by chenyinli — CHANGES_REQUESTED** [review]

**Review by chenyinli — APPROVED** [review]

**Pull request merged**